scripts/upload_master_schedule.py

```diff
@@ -7,28 +7,28 @@
 # For simplicity, this script uses relative paths
 # Thus, it must be run from the scripts folder
 DIRNAME = os.path.dirname(__file__)
-EXCEPTIONS_PATH = os.path.join(DIRNAME, "../data/exceptions.json")
+MASTERSCHED_PATH = os.path.join(DIRNAME, "../data/master_schedule.json")
```

Comment:
Nit: Python frowns upon abbreviations - you should call this variable `MASTER_SCHEDULE_PATH`

Reply:
fixed both things!